basics_dev/coding-style.md

```diff
@@ -165,7 +165,7 @@ Security coding guidelines
        height = untrusted_conf.height;
     ~~~
 
--   Use another variables, without the `untrusted_` prefix to hold the sanitized values, as shown above.
```

Comment:
'Use alternative variables, without the `untrusted_` prefix to hold the sanitized values of the equivalent above.' ?